scripts/tests/secrets-integration.sh

```diff
@@ -75,9 +75,20 @@ EOF
 
 # set and test get returns the correct value
 "$earthly" secrets --org manitou-org --project earthly-core-integration-test set my_test_file "secret-value"
-
 "$earthly" secrets --org manitou-org --project earthly-core-integration-test get my_test_file | acbgrep 'secret-value'
 
+# test set and get with org selected in config file
+"$earthly" org select manitou-org
+"$earthly" secrets --project earthly-core-integration-test get my_test_file | acbgrep 'secret-value'
+"$earthly" secrets --project earthly-core-integration-test set my_other_file "super-secret-value"
+"$earthly" secrets --project earthly-core-integration-test get my_other_file | acbgrep 'super-secret-value'
+
```

Comment:
can you also include some tests to validate the output of `earthly secrets ls`?

Reply:
Yep, will do